### ISSUE TITLE
Linker command failed - library not found for -lPods-Responder

### ISSUE THREAD

**Issue by jrmcdona** [phonegap/phonegap-plugin-push#1748]
My ionic project called Responder compiles fine with:
ionic cordova build ios.

However, it will not compile in xcode due to a Pods linking issue.   Logs are at the bottom.  I tried removing the platform and re-adding like this "ionic cordova platform add  ios@4.4.0" but same issue.

I have no Pods scheme in my Responder.xcodeproj but I do in Responder.xcworkspace project and workspace builds.  I installed cocoapods and ran pod setup.

Any idea what step I am missing?

![screen shot 2017-05-29 at 7 19 27 pm](https://cloud.githubusercontent.com/assets/4327773/26583072/4ab6d42a-4509-11e7-9860-c96fe5ba06f8.png)

#### Cordova CLI version and cordova platform version
global packages:

    @ionic/cli-utils : 1.2.0
    Cordova CLI      : 7.0.1 
    Ionic CLI        : 3.2.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.2.1
    @ionic/cli-plugin-ionic-angular : 1.2.0
    Cordova Platforms               : ios 4.4.0
    Ionic Framework                 : ionic-angular 3.0.1

System:

    Node       : v7.4.0
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.2 Build version 8E2002 
    ios-deploy : 1.9.1 
    ios-sim    : 5.0.12 

#### Plugin version

    cordova plugin version | grep phonegap-plugin-push   # e.g. 1.5.3

#### Logs taken while reproducing problem

Ld /Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Products/Debug-iphonesimulator/Responder.app/Responder normal x86_64
    cd /Users/jordanmc/Documents/src/Responder/Microsoft.Responder/platforms/ios
    export IPHONEOS_DEPLOYMENT_TARGET=9.0
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator10.3.sdk -L/Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Products/Debug-iphonesimulator -L/Users/jordanmc/Documents/src/Responder/Microsoft.Responder/platforms/ios/Pods/GGLInstanceID/Libraries -L/Users/jordanmc/Documents/src/Responder/Microsoft.Responder/platforms/ios/Pods/GoogleCloudMessaging/Libraries -F/Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Products/Debug-iphonesimulator -F/Users/jordanmc/Documents/src/Responder/Microsoft.Responder/platforms/ios/Pods/GoogleIPhoneUtilities/Frameworks -F/Users/jordanmc/Documents/src/Responder/Microsoft.Responder/platforms/ios/Pods/GoogleInterchangeUtilities/Frameworks/frameworks -F/Users/jordanmc/Documents/src/Responder/Microsoft.Responder/platforms/ios/Pods/GoogleSymbolUtilities/Frameworks/frameworks -F/Users/jordanmc/Documents/src/Responder/Microsoft.Responder/platforms/ios/Pods/GoogleUtilities/Frameworks/frameworks -filelist /Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Intermediates/Responder.build/Debug-iphonesimulator/Responder.build/Objects-normal/x86_64/Responder.LinkFileList -Xlinker -rpath -Xlinker @executable_path/Frameworks -mios-simulator-version-min=9.0 -dead_strip -Xlinker -object_path_lto -Xlinker /Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Intermediates/Responder.build/Debug-iphonesimulator/Responder.build/Objects-normal/x86_64/Responder_lto.o -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -ObjC -ObjC -lGGLInstanceIDLib -lGcmLib -lsqlite3 -lz -framework AddressBook -framework CoreGraphics -framework GoogleIPhoneUtilities -framework GoogleInterchangeUtilities -framework GoogleSymbolUtilities -framework GoogleUtilities -framework SystemConfiguration -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker /Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Intermediates/Responder.build/Debug-iphonesimulator/Responder.build/Responder.app.xcent /Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Products/Debug-iphonesimulator/libCordova.a -weak_framework WebKit -lz -framework AddressBook -lsqlite3 -lPods-Responder -Xlinker -dependency_info -Xlinker /Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Intermediates/Responder.build/Debug-iphonesimulator/Responder.build/Objects-normal/x86_64/Responder_dependency_info.dat -o /Users/jordanmc/Library/Developer/Xcode/DerivedData/Responder-gxmtnsbtcukdazexodqiycsumspa/Build/Products/Debug-iphonesimulator/Responder.app/Responder

ld: library not found for -lPods-Responder
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by macdonst** [comment]
@jrmcdona open `Responder.xcworkspace` and build from there. As of Xcode 4 if you see a xcworkspace file you should open that in Xcode to build your project as it is the glue that holds all the sub-projects together.

**Comment by macdonst** [comment]
@jrmcdona if that doesn't work for you please comment on this issue but for now I'm going to close it as I'm 99.44% sure that is your problem.

**Comment by jrmcdona** [comment]
yes, that works and mentioned that in original post.  But it still does not build in .xcodeproj after that. I just tried that again.   Should I stop using .xcodeproject altogether now?

Here are the files that seem to have an issue in the .xcodeproj.   I have cleaned and rebuilt many times.

![screen shot 2017-05-30 at 7 28 48 am](https://cloud.githubusercontent.com/assets/4327773/26583311/54719be8-450a-11e7-892e-100bfc712751.png)
